internal/tools/orchestrator/initialize.go

```diff
@@ -24,6 +24,7 @@ import (
 
 	"github.com/erda-project/erda-infra/base/servicehub"
 	infrahttpserver "github.com/erda-project/erda-infra/providers/httpserver"
+
```

Comment:
reorder your imports. put all dependencies with `github.com/erda-project/` prefix together.

see the rule at `tools/gotools/go-imports-order/main.go`:

```
Imports order must be：

1. Golang Standard Package
2. Third Party Package
3. Erda Project Package
```

But the github actions all passed, so I think maybe we should improve it.